### ISSUE TITLE
create installer artifact

### ISSUE THREAD

**Issue by gogo2464** [rizinorg/rizin#4557]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated because I would like to have the latest version with no rebuild

**Describe the solution you'd like**
adding artifact to github

**Comment by XVilka** [comment]
It is already here: https://ci.appveyor.com/project/rizinorg/rizin/builds/50125190/job/6ewk3740kyarh9r1/artifacts
Just choose: CI statuses for every commit, click AppVeyor one, then choose VS2022 job, then Artifacts tab.

**Comment by XVilka** [comment]
@gogo2464 does it solve your problem?

**Comment by gogo2464** [comment]
yes I just would like to have the path for the future please? :)
